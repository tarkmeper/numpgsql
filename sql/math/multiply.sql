SELECT multiply('{1,2}'::integer[], '{3,4}'::integer[]);

--Single value
SELECT '{{1,2},{3,4}}'::integer[] * 3;
