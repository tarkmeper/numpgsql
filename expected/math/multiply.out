SELECT multiply('{1,2}'::integer[], '{3,4}'::integer[]);
 multiply 
----------
 {3,8}
(1 row)

--Single value
SELECT '{{1,2},{3,4}}'::integer[] * 3;
    ?column?    
----------------
 {{3,6},{9,12}}
(1 row)

